patterns/media-with-text-featured.php

```diff
@@ -7,7 +7,7 @@
 ?>
 <!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"0","bottom":"0"}}},"textColor":"ucsc-primary-blue","className":"ucsc__media-with-text-featured","layout":{"type":"constrained","contentSize":"","wideSize":""}} -->
 <div class="wp-block-group alignfull ucsc__media-with-text-featured has-ucsc-primary-blue-color has-text-color" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0"><!-- wp:media-text {"mediaId":52,"mediaLink":"https://www-next.ucsc.edu/?attachment_id=52","mediaType":"image","mediaWidth":35,"mediaSizeSlug":"large","imageFill":true} -->
-<div class="wp-block-media-text alignwide is-stacked-on-mobile is-image-fill" style="grid-template-columns:35% auto"><figure class="wp-block-media-text__media" style="background-image:url(https://www-next.ucsc.edu/wp-content/uploads/2022/10/20170617-172145-1024x682.jpg);background-position:50% 50%"><img src="https://www-next.ucsc.edu/wp-content/uploads/2022/10/20170617-172145-1024x682.jpg" alt="" class="wp-image-52 size-large"/></figure><div class="wp-block-media-text__content"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}},"fontSize":"eight"} -->
+<div class="wp-block-media-text alignwide is-stacked-on-mobile is-image-fill" style="grid-template-columns:35% auto"><figure class="wp-block-media-text__media"><img alt="" class="wp-image-52 size-large"/></figure><div class="wp-block-media-text__content"><!-- wp:heading {"style":{"spacing":{"margin":{"top":"0","right":"0","bottom":"0","left":"0"}}},"fontSize":"eight"} -->
```

Comment:
@bjcooper  we need to remove those classes too. they're specific to the image ID and size available.